packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -626,7 +631,8 @@ class _BottomSheetSuspendedCurve extends ParametricCurve<double> {
 /// for more details).
 ///
 /// The [transitionAnimationController] controls the bottom sheet's entrance and
-/// exit animations if provided.
+/// exit animations if provided. It's up to the owner of the controller to call
```

Comment:
wordsmithing: It's up to the owner of the controller to call [AnimationController.dispose] when the controller is no longer needed.

Please apply this (trivial) update elsewhere.